flow/test/test_delta_debug.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# deltaDebug.py integration smoke-test, run from ORFS/flow folder.
+#
+# Exit with error if anything is amiss, including evaluation of
+# variable names such as $(false), unused variables, etc.
+set -ue -o pipefail
+
+testname=uart
+
+make DESIGN_CONFIG=designs/asap7/$testname/config.mk place
+make DESIGN_CONFIG=designs/asap7/$testname/config.mk global_place_issue
+latest_file=$(ls -t global_place_${testname}_asap7_base*.tar.gz | head -n1)
+echo "Testing $latest_file"
+. ../env.sh
+rm -rf test_delta_debug
+mkdir test_delta_debug
```

Comment:
Write into test/results as noted in your gitignore PR

Reply:
Fixed.